# Inclusion of EPA analysis for a Single-Step for Reaction-Norms in Multi-environmet GBLUP

**Description:**
> *In developing*. Codes and tutorials related to "Environmental-Phenotype Associations" (EPA) and its uses in multi-environment GBLUP.

**Created at     : 2022-02-02**
**Last update at : 2022-03-23**
 ----------------------------------------------------------------------------------------------------------------
<div id="p2" />

                                   
                   
# Suggested Literature

* Costa-Neto, G, Galli G, Carvalho, H F, Fritsche-Neto R, Crossa J (2020). EnvRtype: A package to interplay enviromics and quantitative genomics. available at [biorxv](https://www.biorxiv.org/content/10.1101/2020.10.14.339705v3)

* Costa-Neto G, Fritsche-Neto R, Crossa J (2020). Nonlinear kernels, dominance, and envirotyping data increase the accuracy of genome-based prediction in multi-environment trials. Heredity (Edinb). http://dx.doi.org/10.1038/s41437-020-00353-1

* Crossa J, Martini JWR, Gianola D, et al (2019) Deep Kernel and Deep Learning for Genome-Based Prediction of Single Traits in Multienvironment Breeding Trials. Front Genet 10:1–13. https://doi.org/10.3389/fgene.2019.01168

* Cuevas J, Montesinos-López O, Juliana P, et al (2019) Deep Kernel for genomic and near infrared predictions in multi-environment breeding trials. G3 Genes, Genomes, Genet 9:2913–2924. https://doi.org/10.1534/g3.119.400493

* Granato I, Cuevas J, Luna-vázquez F, et al (2018) BGGE : A New Package for Genomic-Enabled Prediction Incorporating Genotype x Environment Interaction Models. 8:3039–3047. https://doi.org/10.1534/g3.118.200435

* Morota G, Gianola D (2014) Kernel-based whole-genome prediction of complex traits: A review. Front Genet 5:. https://doi.org/10.3389/fgene.2014.00363

* Pérez-Elizalde S, Cuevas J, Pérez-Rodríguez P, Crossa J (2015) Selection of the Bandwidth Parameter in a Bayesian Kernel Regression Model for Genomic-Enabled Prediction. J Agric Biol Environ Stat 20:512–532. https://doi.org/10.1007/s13253-015-0229-y

* Jarquín D, Crossa J, Lacaze X, et al (2014) A reaction norm model for genomic selection using high-dimensional genomic and environmental data. Theor Appl Genet 127:595–607. https://doi.org/10.1007/s00122-013-2243-1


* Pérez-Rodríguez P, Gianola D, González-Camacho JM, et al (2012) Comparison between linear and non-parametric regression models for genome-enabled prediction in wheat. G3 Genes, Genomes, Genet. https://doi.org/10.1534/g3.112.003665

* VanRaden PM (2008) Efficient methods to compute genomic predictions. J Dairy Sci 91:4414–4423. https://doi.org/10.3168/jds.2007-0980




 ----------------------------------------------------------------------------------------------------------------
<div id="p9" />

# Authorship and Acknowledgments

> Questions, suggestions or corrections should be adressed to t **Germano Costa Neto** <germano.cneto@gmail.com>

> We thank the wheat breeding team rom CIMMYT for their support in the development of the project.

    
----------------------------------------------------------------------------------------------------------------
<div id="p10" />

